cli/dump_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/config"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+)
+
+func TestDBRestore(t *testing.T) {
+	tmpDir := os.TempDir()
+	chainPath := path.Join(tmpDir, "neogotestchain")
+	defer os.RemoveAll(chainPath)
+	cfg, err := config.LoadFile("../config/protocol.unit_testnet.yml")
+	require.NoError(t, err, "could not load config")
+	cfg.ApplicationConfiguration.DBConfiguration.Type = "leveldb"
+	cfg.ApplicationConfiguration.DBConfiguration.LevelDBOptions.DataDirectoryPath = chainPath
+
+	out, err := yaml.Marshal(cfg)
+	require.NoError(t, err)
+
+	cfgPath := path.Join(tmpDir, "protocol.unit_testnet.yml")
+	require.NoError(t, ioutil.WriteFile(cfgPath, out, os.ModePerm))
+	defer os.Remove(cfgPath)
+
+	stateDump := path.Join(tmpDir, "neogo.teststate")
+	defer os.RemoveAll(stateDump)
+
+	const inDump = "./testdata/chain50x2.acc"
```

Comment:
This dump will need to be updated in case anything changes in the protocol, so you need to explain how to recreate it.

Reply:
Fixed.